frontend/src/Redux/reducers.js

```diff
@@ -3,45 +3,11 @@ import houseIssues from '../Dashboard/reducers';
 import house from '../House/reducers';
 import issue from '../Issue/reducers';
 import search from '../Search/reducers';
-import * as UserTypes from './UserTypes';
-
-function reducer(state = {user:{}, isWorking:false, error:""}, action) {
-  switch (action.type) {
-    case UserTypes.AUTO_LOGIN_SUCCESS:
-    case UserTypes.REQUEST_LOGIN_SUCCESS:
-      return {...state,
-        user: action.user
-      };
-    case UserTypes.AUTO_LOGIN_FAILED:
-    case UserTypes.REQUEST_LOGIN_FAILED:
-      return {...state,
-        error: action.error
-      };
-    case UserTypes.USER_IS_WORKING:
-      return {...state,
-        isWorking: action.isWorking
-      };
-    case UserTypes.USER_SUCCESS:
-      return {...state,
-        user: action.user
-      };
-    case UserTypes.USER_ERRORED:
-      return {
-        error: action.userErrored
-      };
-    case UserTypes.CLEAR_LOGIN_ERROR:
-    case UserTypes.LOGOUT:
-      return {...state,
-        user: {}
-      };
-    default:
-      return state;
-  }
-}
+import userReducer from './userReducer';
 
 // user key has to be user
 export default combineReducers({
-  user: reducer,
+  user: userReducer,
```

Comment:
Do we need this inconsistency? can't we just have `userReducer` or `user` to follow the suit of the others?